test/unittests/libweb3jsonrpc/jsonrpc.cpp

```diff
@@ -669,8 +691,8 @@ BOOST_AUTO_TEST_CASE( eth_sendRawTransaction_errorInvalidNonce,
     signedTx = fixture.rpcClient->eth_signTransaction( t );
     BOOST_REQUIRE( !signedTx["raw"].empty() );
 
-    BOOST_CHECK_EQUAL(
-        fixture.sendingRawShouldFail( signedTx["raw"].asString() ), "Invalid transaction nonce." );
+
+    fixture.sendingRawShouldFail( signedTx["raw"].asString() );

```

Comment:
It's better to check the error message as it was before

Reply:
Fixed